include/mbedtls/ssl_internal.h

```diff
@@ -786,4 +786,14 @@ int mbedtls_ssl_get_key_exchange_md_tls1_2( mbedtls_ssl_context *ssl,
 }
 #endif
 
+void ssl_transform_init( mbedtls_ssl_transform *transform );
```

Comment:
Doesn't `check-names.sh` complain about this? I think it should. Even though this is an internal header, non-static functions will still be exposed in the library, and need to avoid name clashes with other libraries the program might be linking in, which is best achieved by using the `mbedtls_` prefix consistently.